subsys/net/l2/wifi/wifi_utils.c

```diff
@@ -63,3 +63,40 @@ int wifi_utils_parse_scan_bands(char *scan_bands_str, uint8_t *band_map)
 
 	return 0;
 }
+
+int wifi_utils_parse_scan_ssids(char *scan_ssids_str,
+				char ssids[][WIFI_SSID_MAX_LEN])
+{
+	char *ssid = NULL;
+	char *ctx = NULL;
+	uint8_t i = 0;
+
+	if (!scan_ssids_str) {
+		return -EINVAL;
+	}
+
+	ssid = strtok_r(scan_ssids_str, ",", &ctx);
+
+	while (ssid) {
+		if (strlen(ssid) > WIFI_SSID_MAX_LEN) {
```

Comment:
`WIFI_SSID_MAX_LEN` this doesn't include the NULL de-limiter, so, need to add `+ 1`

Reply:
This check is still valid since strlen does not consider the NULL terminator.